### PR TITLE
UI/fix kmip role form

### PR DESCRIPTION
**Pull request by hashishaw**
Before this fix, there was a bug where creating a KMIP role did not save any non-operator keys. There was also an issue where TLS info was not displayed on the show details page, due to an issue in InfoTableRow. As part of this work, I glimmerized InfoTableRow Component and added tests for various scenarios.

**Before (key bits value does not save)**
bad-role

**After**
kmip-role-after

### CODE REVIEW

File: ui/app/templates/components/identity/item-alias/alias-details.hbs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<InfoTableRow @label="Name" @value={{@model.name}} @data-test-alias-name={{true}} />
+<InfoTableRow @label="Name" @value={{@model.name}} data-test-alias-name={{true}} />
```
Comment (Monkeychip) [inline ui/app/templates/components/identity/item-alias/alias-details.hbs:1]: Curious how that @data.... got in there!

Comment (Monkeychip) [reply ui/app/templates/components/identity/item-alias/alias-details.hbs:1]: Probably an upgrade maybe even back to the big one in 1.7

Comment (hashishaw) [reply ui/app/templates/components/identity/item-alias/alias-details.hbs:1]: I think it was the latest ember upgrade. I considered find-replacing all the `@data-test` instances, but we have a few cases where it looks like they're actually supposed to work that way (eg. ToolbarSecretLink 😱)

File: ui/tests/integration/components/info-table-row-test.js [hunk 2]
```diff
@@ -156,10 +156,96 @@ module('Integration | Component | InfoTableRow', function (hooks) {
...
       @alwaysRender={{true}}>
-      Block content is here 
+      Block content is here
       </InfoTableRow>`);
 
     let block = document.querySelector('[data-test-value-div]').textContent.trim();
     assert.equal(block, 'Block content is here', 'renders block passed through');
   });
+
+  test('Row renders when block content even if alwaysRender = false', async function (assert) {
```
Comment (Monkeychip) [inline ui/tests/integration/components/info-table-row-test.js:166]: Thank you for adding test coverage!

**Review by Monkeychip — APPROVED** [review]
Nice work!

**Comment by zofskeez** [comment]
I was wondering how the layout thing was going to work when glimmerizing addon components. Looks like the template has to be moved along side the component file and it just works?

**Pull request merged**